pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/codestyle/CommentDefaultAccessModifierRule.java

```diff
@@ -41,6 +41,14 @@
 
     public CommentDefaultAccessModifierRule() {
         definePropertyDescriptor(REGEX_DESCRIPTOR);
+        defaultSuppressionAnnotations();
+    }
+
+    @Override
+    protected Collection<String> defaultSuppressionAnnotations() {
+        Collection<String> ignoredStrings = new ArrayList<>();
+        ignoredStrings.add("VisibleForTesting");
```

Comment:
you should use fully qualified class names where for proper type matching.

the ones we probably care about are:
`com.google.common.annotations.VisibleForTesting`
`android.support.annotation.VisibleForTesting`